src/_sass/_secondary_nav.scss

```diff
@@ -0,0 +1,196 @@
+    width: 0; 
+  }
+
+  .sub-nav-control {
+    margin-left: 0.625rem;
+    transition: linear .2s;
+  }
+
+
+  .sub-nav-control:hover {
+    cursor: pointer;
+  }
+
+  .hello {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    
+    // .secondary-nav-link {
+    //   color: $secondary-nav-color;
+    //   font-family: "Open Sans";
+    //   font-size: 1rem;
+    //   letter-spacing: -0.0125rem;
+    //   line-height: 1rem;
+    //   padding-bottom: 0.5rem;
+    //   text-decoration: none;
+    // }
+
+    .secondary-nav-link {
+      position: relative;
+      color: white;
+      text-decoration: none;
+    }
+    
+    .secondary-nav-link:before {
+      content: "";
+      position: absolute;
+      width: 100%;
+      height: 2px;
+      bottom: -10px;
+      left: 0;
+      background-color: white;
+      visibility: hidden;
+      -webkit-transform: scaleX(0);
+      transform: scaleX(0);
+      -webkit-transition: all 0.3s ease-in-out 0s;
+      transition: all 0.3s ease-in-out 0s;
+    }
+    
+    .secondary-nav-link:hover {
+      cursor: pointer;
+    }
+    
+    .secondary-nav-link:hover:before {
+        visibility: visible;
+        transform: scaleX(1);
+    }
+
+    ul {
+      display: flex;
+      justify-content: space-between;
+      li {
+        margin-left: 1.875rem;
+      }
+      .current:before {
+        visibility: visible;
+        transform: scaleX(1);
+      }
+    }
+    
+  }
+
+  .current-active {
+    color: $current-color;
+  }
+
+  .content:after {
+    content: none;
+  }
+
+  .mobile {
+    background-color: #f9f9f9;
+    box-shadow: 0rem 0.75rem 3.5rem -0.75rem rgba(0, 0, 0, 0.43);
+    left: 0;
+    position: absolute;
+    top: -18.75rem;
+    width: 100%;
+    z-index: -1;
+
+    li {
+      padding: 0.9375rem 0;
+      transition: background-color .2s ease-out 0s;
+    }
+
+    li:hover {
+      background: #e45b6b;
+      .secondary-nav-link {
+        color: #f9f9f9;
+      };  
+    }
+
+    @mixin animation_delay($seconds) {
+      transition: 
+        color .5s ease 0s,
+        margin .3s cubic-bezier(0.74,-0.21, 0.47, 1.46) $seconds,
+        opacity .3s ease 0.01s;
+    }
+
+    li:nth-child(1) .secondary-nav-link {
+      @include animation_delay(.01s)
+    }
+
+    li:nth-child(2) .secondary-nav-link {
+      @include animation_delay(.06s)
+    }
+    li:nth-child(3) .secondary-nav-link {
+      @include animation_delay(.12s)
+    }
+    li:nth-child(4) .secondary-nav-link {
+      @include animation_delay(.18s)
+    }
+    li:nth-child(5) .secondary-nav-link {
+      @include animation_delay(.24s)
+    }
+    li:nth-child(6) .secondary-nav-link {
+      @include animation_delay(.30s)
+    }
+
+    .secondary-nav-link {
+      color: #e45b6b;
+      display: block;
+      font-family: "Open Sans";
+      font-size: 1rem;
+      height: 100%;
+      letter-spacing: -0.0125rem;
+      line-height: 1rem;
+      margin-left: -3.125rem;
+      margin: 0;
```

Comment:
this overwrites the margin-left declaration below. if you want margin: 0, delete margin-left